openpype/hosts/nuke/plugins/load/load_image.py

```diff
@@ -1,6 +1,6 @@
 import nuke
 
-import qargparse
+from openpype.vendor.python.common import qargparse
```

Comment:
```suggestion
import qargparse
```

Reply:
Is this the right file to modify ? The error above seems to come from openpype\tools\utils\widgets.py

Reply:
Both should stay as `import argparse`.